### PR TITLE
Added Terminal support for `%override`

### PR DESCRIPTION
**Pull request by MegaIng**
First point in #803

### CODE REVIEW

**Comment by MegaIng** [comment]
`lark.lark` update in #805

File: lark/load_grammar.py [hunk 1]
```diff
@@ -998,9 +999,15 @@ def load_grammar(self, grammar_text, grammar_name='<?>', import_paths=[]):
...
                 for t in stmt.children:
                     term_defs.append([t.value, (None, None)])
-            elif stmt.data == 'override_rule':
+            elif stmt.data == 'override':
                 r ,= stmt.children
-                overriding_rules.append(options_from_rule(*r.children))
+                if r.data == 'rule':
+                    overriding_rules.append(options_from_rule(*r.children))
+                else:
+                    if len(r.children) == 2:
```
Comment (erezsh) [inline lark/load_grammar.py:1007]: These 4 lines should be a extracted into a function

File: docs/grammar.md [hunk 2]
```diff
@@ -291,7 +291,7 @@ Declare a terminal without defining it. Useful for plugins.
 
 ### %override
 
-Override a rule, affecting all the rules that refer to it.
+Override a rule or terminals, affecting all references to it, even in imported grammars.
```
Comment (erezsh) [inline docs/grammar.md:294]: terminal, singular.

**Review by erezsh — CHANGES_REQUESTED** [review]

**Comment by erezsh** [comment]
Outdated

**Pull request closed without merging**